doc/nrf/ug_nrf52.rst

```diff
@@ -0,0 +1,95 @@
+
+For example, for the :ref:`nRF52840 Development Kit (PCA10056) <zephyr:nrf52840_pca10056>`, use ``nrf52840_pca10056`` as board name when building.
+
+Secure bootloader chain
+=======================
+
+nRF52 Series devices support a secure bootloader solution based on the chain of trust concept.
+
+See :ref:`ug_bootloader` for more information and instructions on how to enable one or more bootloaders in your application.
+
+
+Supported protocols
+*******************
+
+The main area of application for the nRF52 Series is Bluetooth Low Energy, but you can also use the proprietary ESB protocol, NFC, or USB.
+The following sections give pointers on where to start when working with these protocols.
+
+To test the general capabilities of the 2.4 GHz radio transceiver, use the :ref:`radio_test` sample.
+
+Bluetooth Low Energy
+====================
+
+When you develop a Bluetooth Low Energy (LE) application, you must use the Bluetooth LE stack.
+This stack is split into two different layers: the Bluetooth LE Host and the Bluetooth LE Controller.
+
+The |NCS| uses Zephyr's implementation of the Bluetooth LE Host.
+For the Bluetooth LE Controller, you can choose between the nRF Bluetooth LE Controller that is implemented by Nordic Semiconductor and Zephyr's open-source Bluetooth LE Controller.
+The :ref:`ug_ble_controller` user guide contains more information about the two available controllers and instructions for switching between them.
+
+See :ref:`zephyr:bluetooth` for documentation on Zephyr's Bluetooth LE stack, including both host and controller.
+For documentation about the nRF Bluetooth LE Controller and information on what variants of the controller support with chips, see :ref:`nrfxlib:ble_controller`.
+
+The |NCS| contains a variety of :ref:`ble_samples` that target nRF52 Series devices.
+In addition, you can run any of Zephyr's :ref:`zephyr:bluetooth-samples`.
+
+For available libraries, see :ref:`lib_bluetooth_services` (|NCS|) and :ref:`zephyr:bluetooth_api` (Zephyr).
+
+
+Enhanced ShockBurst
+===================
+
+.. include:: ug_esb.rst
+   :start-after: esb_intro_start
+   :end-before: esb_intro_end
+
+See the :ref:`ug_esb` user guide for information about how to work with Enhanced ShockBurst.
+To start developing, check out the :ref:`esb_prx_ptx` sample.
+
+Near field communication
+========================
+
+Near field communication (NFC) is a technology for wireless transfer of small amounts of data between two devices.
+It uses very simple communication protocols and therefore allows quicker connection than Bluetooth LE.
+However, because NFC uses magnetic induction to enable communication, the devices must be very close (< 10 cm) to connect.
+
+The stack for developing NFC applications is provided in binary format in the `nrfxlib`_ repository.
+See :ref:`nrfxlib:nfc` for documentation about the NFC stack.
+
+The NFC stack requires the NFCT driver for nRF52 devices, which is available as part of `nrfx`_.
```

Comment:
The "nrfx" just links out to git hub.  That gives the impression the user needs to do something to use it.  I would prefer we link to a new page, included at the top level "About the nRF Connect SDK" which explains that nrfx contains Nordic's RTOS independent driver implementations, it is already incorporated in the Zephyr repository and is available separately in github <link>."

Reply:
Will add a sentence that nrfx is included through the Zephyr repo. The other updates should come as part of including the nrfx documentation.